### PR TITLE
docs: fixed invalid URLs in translation docs

### LINKED ISSUE #3791: invalid URLs in translation docs
# Improve Docs

## Please describe which part of docs should be improved or typo fixed

Hi, we have contributors who noticed that there have some broken links in translated docs:

e.g.: https://apisix.apache.org/zh/docs/apisix/discovery/
and this doc is targeted to https://github.com/apache/apisix/edit/master/docs/zh/latest/discovery.md

image

## Describe the solution you'd like

Those broken links are due to our doc site doesn't support relative reference yet, for example:

image

In this file, there have 2 links which are targeted the outside files. Here is the solution to fix this issue:

For those referenced links, please use `https://github.com/apache/apisix/blob/master/docs/en/latest/{FILE_PATH}` instead, e.g: `https://github.com/apache/apisix/blob/master/docs/en/latest/discovery/dns.md`

### PR DESCRIPTION
**Pull request by iamayushdas**
### What this PR does / why we need it:

Changing the URL.

Fixed #3791 
### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by juzhiyuan — APPROVED** [review]
LGTM, and there have some other cases like L72

image

We could fix them in another PR :)

**Comment by iamayushdas** [comment]
> LGTM, and there have some other cases like L72
> 
> image
> 
> We could fix them in another PR :)

but It is working properly in this state. do i still have to change it?

**Review by iamayushdas — COMMENTED** [review]
@juzhiyuan can you please re trigger travisCI

**Comment by juzhiyuan** [comment]
> > LGTM, and there have some other cases like L72
> > image
> > We could fix them in another PR :)
> 
> but It is working properly in this state. do i still have to change it?

Hi, you could have a try here: https://apisix.apache.org/zh/docs/apisix/discovery/#%E4%BB%A5-eureka-%E4%B8%BE%E4%BE%8B

File: docs/zh/latest/discovery.md [hunk 1]
```diff
@@ -49,9 +49,9 @@ title: 集成服务发现注册中心
 
 目前支持 Eureka / Consul 和基于 DNS 的服务注册发现。
 
-基于 DNS 的服务注册发现见 [基于 DNS 的服务支持发现](../../en/latest/discovery/dns.md)。
+基于 DNS 的服务注册发现见 [基于 DNS 的服务支持发现](https://github.com/apache/apisix/blob/master/docs/en/latest/discovery/dns.md)。
```
[the remark this answers is not in the document]
Comment (juzhiyuan) [reply docs/zh/latest/discovery.md:52]: yes, when this framework building, broken links occurred.

[the remark this answers is not in the document]
Comment (juzhiyuan) [reply docs/zh/latest/discovery.md:52]: or @qier222, please check this issue to see if there have solutions to support this case.

[the remark this answers is not in the document]
Comment (qier222) [reply docs/zh/latest/discovery.md:52]: We can modify `sync-docs.js` to replace URLs like this.

[the remark this answers is not in the document]
Comment (spacewander) [reply docs/zh/latest/discovery.md:52]: The current solution won't work if we introduce multiple versions of docs.

[the remark this answers is not in the document]
Comment (juzhiyuan) [reply docs/zh/latest/discovery.md:52]: ok 🤔 @qier222 would you have time to fix this issue? @iamayushdas and thanks for your contribution, but please be patient to wait for @qier222's update, he has better solutions.

[the remark this answers is not in the document]
Comment (qier222) [reply docs/zh/latest/discovery.md:52]: I am available tomorrow or the day after tomorrow

[the remark this answers is not in the document]
Comment (juzhiyuan) [reply docs/zh/latest/discovery.md:52]: no problem

**Comment by juzhiyuan** [comment]
https://github.com/apache/apisix-website/pull/257

Hi, this PR fixes most issues, we could close this PR IMO.

cc @iamayushdas Thanks for your watch and contribution!!

**Pull request closed without merging**